domenetjenester/arbeidsforhold/src/test/java/no/nav/foreldrepenger/domene/arbeidsforhold/impl/InntektsmeldingRegisterTjenesteSvpTest.java

```diff
@@ -435,8 +502,108 @@ void ingen_manglende_inntektsmeldinger() {
+        var yrkesaktivitet2 = lagYrkesaktivitetBuilder(List.of(ansettelse2, arbavtale2), arbeidsgiver, ref2, List.of(permisjon2));
+
+        lagArbeid(List.of(yrkesaktivitet1, yrkesaktivitet2));
+        lagInntekt(arbeidsgiver, SKJÆRINGSTIDSPUNKT.minusMonths(12), 12);
+        var inntektsmeldingMedArbId = lagInntektsmelding(arbeidsgiver, BigDecimal.valueOf(55000), EksternArbeidsforholdRef.ref("1"), ref);
+        var grunnlag = byggIAY(List.of(inntektsmeldingMedArbId));
+
+        var tilrFomArbeidsforhold1 = lagTilretteleggingFom(SKJÆRINGSTIDSPUNKT, TilretteleggingType.DELVIS_TILRETTELEGGING);
+        var tilretteleggingEntitet = lagTilrettelegging(arbeidsgiver, null, SKJÆRINGSTIDSPUNKT, List.of(tilrFomArbeidsforhold1));
+
+        var svpGrunnlag = byggSvangerskapspengerGrunnlag(List.of(tilretteleggingEntitet), behandlingReferanse.behandlingId());
+
+        when(svangerskapspengerRepository.hentGrunnlag(behandlingReferanse.behandlingId())).thenReturn(Optional.of(svpGrunnlag));
+        when(inntektArbeidYtelseTjeneste.finnGrunnlag(behandlingReferanse.behandlingId())).thenReturn(Optional.of(grunnlag));
+        when(inntektsmeldingTjeneste.hentInntektsmeldinger(behandlingReferanse, skjæringstidspunkt.getUtledetSkjæringstidspunkt())).thenReturn(
+            List.of(inntektsmeldingMedArbId));
+
+        var manglendeInntektsmeldinger = inntektsmeldingRegisterTjeneste.utledManglendeInntektsmeldingerFraGrunnlag(behandlingReferanse, skjæringstidspunkt);
+        var statusPerArbeidsgiver = finnStatusForInntektsmeldingArbeidsforhold(behandlingReferanse, skjæringstidspunkt);
 
         assertThat(manglendeInntektsmeldinger).isEmpty();
+        assertThat(statusPerArbeidsgiver).hasSize(1);
+        assertThat(statusPerArbeidsgiver.stream().filter(status -> status.arbeidsgiver().equals(arbeidsgiver))
+            .map(ArbeidsforholdInntektsmeldingStatus::inntektsmeldingStatus)).containsAll(
+            Collections.singleton(ArbeidsforholdInntektsmeldingStatus.InntektsmeldingStatus.MOTTATT));
+    }
+
+    @Test
+    void dersom_søkt_på_inaktivt_arbeidsforhold_svp_skal_vi_kreve_inntektsmelding() {
+        var skjæringstidspunkt = Skjæringstidspunkt.builder().medUtledetSkjæringstidspunkt(SKJÆRINGSTIDSPUNKT).build();
+        var arbeidsgiver = Arbeidsgiver.virksomhet("123456789");
+        var arbeidsgiverUtenInntekt =Arbeidsgiver.virksomhet("987456321");
+        var ansettelse1 = lagAktivitetsAvtaleBuilder(SKJÆRINGSTIDSPUNKT.minusYears(1), null);
+        var ansettelse2 = lagAktivitetsAvtaleBuilder(SKJÆRINGSTIDSPUNKT.minusYears(1), null);
+        var arbavtale1 = lagAktivitetsAvtale100ProsentBuilder(SKJÆRINGSTIDSPUNKT.minusYears(1), null);
+        var arbavtale2 = lagAktivitetsAvtale100ProsentBuilder(SKJÆRINGSTIDSPUNKT.minusYears(1), null);
+        var permisjon2 = byggPermisjon(SKJÆRINGSTIDSPUNKT.minusDays(2), SKJÆRINGSTIDSPUNKT.plusMonths(2),
```

Comment:
2 ting: 
Klarer du deg ikke med 1 arbeidsgiver / arbeidsforhold? 
Bør du ikke ha 100% permisjon i testen - evt 1 test for hver av 50% / 100%?

Reply:
jeg har en 100% og en 50%, og har ryddet litt fordi 50% permisjon lå i mange av testene og det var ikke nødvendig. Fjernet også ett av arbeidsforholdene på en av testene